### ISSUE TITLE
[TODO] Add a Twitter Feed

### ISSUE THREAD

**Issue by odysseywestra** [mypaint/mypaint.github.io#10]
I know twitter is MyPaint main way of communicating to the community is though it's twitter feed. So were would we want to integrate the twitter feed at on the website. Since according #4 the design will be that of resembling the gimp website or Krita website. So where do we want place it?

**Comment by odysseywestra** [comment]
I found this [plugin] https://github.com/scossar/discourse-twitter-stream that enable you to use the twitter stream in header of the discourse site which got me thinking. Discourse dose allow you to create a custom html/css header on top of the header it has. So if we could embed the twitter stream as a drop down menu in the header of mypaint.org, we could do the same for the discourse site. That way people can keep up with happening on twitter whether they are on mypaint.org or the discourse site.

**Comment by urielo515** [comment]
hey there, i know this issue is a little old, but is it still pending? i could try and give it a try.

**Comment by odysseywestra** [comment]
Yeah, you're more than welcome to. Just submit a PR and we can review it.
